### PR TITLE
LRQA-73647 | master

### PR DESCRIPTION
**Pull request by weller0246**
[LRQA-73647](https://issues.liferay.com/browse/LRQA-73647)

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:staging:nocompile

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
@gaborlovasdotcom please review my PR and forward it if you have the time!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase [hunk 1]
```diff
@@ -1077,10 +1077,10 @@ definition {
 	@description = "This is a use case for LPS-131578. User can configure the portal to exclude displayed contents from publishing."
 	@priority = "4"
 	test PublishViaPortletWithoutContent {
-		ApplicationsMenu.gotoPortlet(
-			category = "Configuration",
-			panel = "Control Panel",
-			portlet = "System Settings");
+		property portal.upstream = "marcell";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase:1080]: Please remove this line :).

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Please clean up the commit, and it's good to go

**Comment by weller0246** [comment]
ci:forward

**Pull request closed without merging**